tools/meson.build

```diff
@@ -0,0 +1,29 @@
+#  Copyright © 2022 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+inc = include_directories('../include')
+
+lib_m = cc.find_library('m')
```

Comment:
Since you use this multiple times, maybe you'd prefer to define it in the root meson.build and reuse the found library instead of searching again for it?